Sources/NostrSDK/Tag.swift

```diff
@@ -156,6 +162,8 @@ public class Tag: Codable, Equatable {
             name = .subject
         case "emoji":
             name = .emoji
+        case "t":
```

Comment:
We should actually get rid of the concept of mapping tag names to enums as that model does not actually match reality. For example, the `r` tag can be reference or relay. I think we should just expose it as a raw string. I didn't want to make that change in this PR as it adds too much complexity and falls outside of the scope of what I was trying to implement.

Reply:
I think I agree. This is a big bummer though and we should maybe consider a couple alternatives. You were right to not add that to this PR tho.

Reply:
This isn't a well thought through idea, but maybe each event kind can define their own set of known named tags somehow.